sources/Outputs/src/Diagram.cpp

```diff
@@ -0,0 +1,87 @@
+//
+// Copyright (c) 2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+/**
+ * @file  Diagram.cpp
+ *
+ * @brief Dynaflow launcher Diagram file writer implementation file
+ *
+ */
+
+#include "Diagram.h"
+
+#include "Constants.h"
+
+#include <fstream>
+
+namespace dfl {
+namespace outputs {
+
+Diagram::Diagram(DiagramDefinition&& def) : def_{std::forward<DiagramDefinition>(def)} {}
+
+void
+Diagram::write() {
+  //We don't want to create the file if there are no generators diagrams to write
+  //maybe use stdbuf for using << and not .append(..()))
+  std::stringstream buffer;
+  buffer << "#1";
```

Comment:
add a comment to precise the meaning of 1 here

Reply:
You mean that modelica absolutely requires it ?

Reply:
I don't know the format of the file and there is no documentation at the moment on it, so add a comment to give the information to a developer